### ISSUE TITLE
Whitespace between commands

### ISSUE THREAD

**Issue by cmeury** [roboll/helmfile#67]
We should add a couple of new-lines to improve readability of the output. It's difficult to parse the output quickly:

* where does one release end and another start
* what's helmfile output and helm output

**Comment by cmeury** [comment]
True. How about we keep this issue for the text output and potentially add other formats #68 for programmatic consumption?

**Comment by mumoshu** [comment]
Sounds good 👍
